src/plugin_discovery/find_plugin_specs.js

```diff
@@ -173,8 +197,7 @@ export function findPluginSpecs(settings, config = defaultConfig(settings)) {
     // the config service we extended with all of the plugin specs,
     // only emitted once it is fully extended by all
     extendedConfig$: extendConfig$
-      .ignoreElements()
-      .concat([config]),
+      .mergeMap(result => result.config),
```

Comment:
We should probably include `.filter(Boolean).last()` here, and might as well add a test that verifies that a single `config` instance is emitted and that it has the extensions from all plugins when `emitted`.